seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -695,11 +695,12 @@
       "title": "Import a Google Font",
       "description": [
         "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
-        "First, you'll need to make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
-        "Copy the following code snippet and paste it into the top of your code editor:",
+        "<a href='https://fonts.google.com/' target='_blank'>Google Fonts</a> is a free library of web-safe fonts that you can use in your CSS by referencing the font's URL.",
```

Comment:
@HKuz Great work! I made a mistake though. The fonts on Google Fonts aren't web-safe (my poor understanding of the terminology) so can you change "web-safe" to "web"